doc/prop.xml

```diff
@@ -131,6 +131,40 @@ true
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="IsPreorderDigraph">
+<ManSection>
+  <Prop Name="IsPreorderDigraph" Arg="digraph"/>
+  <Prop Name="IsQuasiorderDigraph" Arg="digraph"/>  
+  <Returns><K>true</K> or <K>false</K>.</Returns>
+  <Description>
+    These are synonyms for <Ref Prop="IsReflexiveDigraph"/> and <Ref
+      Prop="IsTransitiveDigraph"/>.  For a preorder digraph (or quasiorder
+    digraph) <A>digraph</A>, the corresponding preorder is the relation
+    <M>\leq</M>, defined by <M>x \leq y</M> if and only if <C>[x, y]</C> is an
+    edge of <A>digraph</A>.
+
```

Comment:
For consistency with elsewhere in the manual, please write the definition of a preorder explicitly. I.e. a digraph is a <E>preorder</E> if ...

Reply:
In line 141 I write: 
"These are synonyms for <Ref Prop="IsReflexiveDigraph"/> and <Ref Prop="IsTransitiveDigraph"/>."
Is that sufficient since these two properties have manual entries describing exactly what they are? I would just be repeating these two definitions.